[2024-01-10T18:54:53.625+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: mysql_test_2.send_slack manual__2024-01-10T18:49:39.150931+00:00 [queued]>
[2024-01-10T18:54:53.645+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: mysql_test_2.send_slack manual__2024-01-10T18:49:39.150931+00:00 [queued]>
[2024-01-10T18:54:53.646+0000] {taskinstance.py:1279} INFO - 
--------------------------------------------------------------------------------
[2024-01-10T18:54:53.647+0000] {taskinstance.py:1280} INFO - Starting attempt 2 of 2
[2024-01-10T18:54:53.647+0000] {taskinstance.py:1281} INFO - 
--------------------------------------------------------------------------------
[2024-01-10T18:54:53.676+0000] {taskinstance.py:1300} INFO - Executing <Task(SlackWebhookOperator): send_slack> on 2024-01-10 18:49:39.150931+00:00
[2024-01-10T18:54:53.682+0000] {standard_task_runner.py:55} INFO - Started process 980 to run task
[2024-01-10T18:54:53.687+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'mysql_test_2', 'send_slack', 'manual__2024-01-10T18:49:39.150931+00:00', '--job-id', '33', '--raw', '--subdir', 'DAGS_FOLDER/mysql_test_2.py', '--cfg-path', '/tmp/tmpgkwhp0a2']
[2024-01-10T18:54:53.688+0000] {standard_task_runner.py:83} INFO - Job 33: Subtask send_slack
[2024-01-10T18:54:53.806+0000] {task_command.py:388} INFO - Running <TaskInstance: mysql_test_2.send_slack manual__2024-01-10T18:49:39.150931+00:00 [running]> on host 1064147bcaa7
[2024-01-10T18:54:53.955+0000] {taskinstance.py:1509} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=mysql_test_2
AIRFLOW_CTX_TASK_ID=send_slack
AIRFLOW_CTX_EXECUTION_DATE=2024-01-10T18:49:39.150931+00:00
AIRFLOW_CTX_TRY_NUMBER=2
AIRFLOW_CTX_DAG_RUN_ID=manual__2024-01-10T18:49:39.150931+00:00
[2024-01-10T18:54:53.957+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/providers/slack/operators/slack_webhook.py:173: UserWarning: Found unexpected keyword-argument(s) 'link_names' in `send` method. This argument(s) have no effect.
  link_names=self.link_names,

[2024-01-10T18:54:53.973+0000] {base.py:73} INFO - Using connection ID 'slack_conn' for task execution.
[2024-01-10T18:54:53.976+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/providers/slack/hooks/slack_webhook.py:226: DeprecationWarning: Found Slack Webhook Token URL in Connection 'slack_conn' `host` and `password` field is empty. This behaviour deprecated and could expose you token in the UI and will be removed in a future releases.
  return WebhookClient(**self._get_conn_params())

[2024-01-10T18:54:54.528+0000] {taskinstance.py:1323} INFO - Marking task as SUCCESS. dag_id=mysql_test_2, task_id=send_slack, execution_date=20240110T184939, start_date=20240110T185453, end_date=20240110T185454
[2024-01-10T18:54:54.622+0000] {local_task_job.py:208} INFO - Task exited with return code 0
[2024-01-10T18:54:54.718+0000] {taskinstance.py:2578} INFO - 0 downstream tasks scheduled from follow-on schedule check
